### PR TITLE
Remove leftover RBF code from BTC

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Removes leftover RBF code from BTC, it doesn't seem like this will create conflicts on backports. If it would create conflicts we can go ahead and close this PR

### CODE REVIEW

**Comment by UdjinM6** [comment]
There are also few parts in `AcceptToMemoryPoolWorker`, see `setConflicts`, `allConflicting` etc.

**Comment by PastaPastaPasta** [comment]
Builds without error, please re-review @UdjinM6

**Comment by PastaPastaPasta** [comment]
@UdjinM6 comments resolved

File: src/validation.cpp [hunk 1]
```diff
@@ -700,49 +697,21 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
...
+                return state.DoS(0, error("AcceptToMemoryPool : Transaction %s conflicts with Transaction Lock Request %s",
+                                        hash.ToString(), ptxConflicting->GetHash().ToString()),
+                                REJECT_INVALID, "tx-txlockreq-mempool-conflict");
+            } else if (instantsend.HasTxLockRequest(hash)) {
+                // this tx is a tx lock request and it conflicts with a normal tx
+                return state.DoS(0, error("AcceptToMemoryPool : Transaction Lock Request %s conflicts with transaction %s",
+                                        hash.ToString(), ptxConflicting->GetHash().ToString()),
+                                REJECT_INVALID, "txlockreq-tx-mempool-conflict");
             }
+            // Transaction conflicts with mempool and RBF doesn't exsist in Dash
```
Comment (nmarley) [inline src/validation.cpp]: nit: s/exsist/exist/

File: src/net_processing.cpp [hunk 2]
```diff
@@ -2007,7 +2007,7 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
 
         std::list<CTransactionRef> lRemovedTxn;
```
Comment (nmarley) [inline src/net_processing.cpp]: @PastaPastaPasta Looks like this unused list can be removed now.

**Comment by PastaPastaPasta** [comment]
Resolved @nmarley comments

**Pull request merged**